Tests/EventListener/SecurityListenerTest.php

```diff
@@ -26,20 +26,24 @@ class SecurityListenerTest extends \PHPUnit_Framework_TestCase
      */
     public function testLegacyAccessDenied()
     {
-        $this->iniSet('error_reporting', -1 & ~E_USER_DEPRECATED);
+        if (!interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')) {
+            $this->markTestSkipped();
```

Comment:
Indeed, I'd misunderstood the test's purpose.
Reverting and skipping the test when the SecurityContextInterface is not declared. Is that ok?